src/components/SidebarButton.js

```diff
@@ -27,18 +27,21 @@ const ariaMap = {
 };
 
 const SidebarButton = ({ children, icon, onClick, expanded }) => (
-  <button
-    className={classNames('SidebarButton', {
-      'SidebarButton-expanded': expanded
-    })}
-    onClick={onClick}
-    aria-label={ariaMap[icon]}
-    aria-expanded={expanded}
-    data-testid={`SidebarButton-${icon}-button`}
-  >
-    <img src={iconMap[icon]} alt={`${icon} icon`} />
-    {children}
-  </button>
+  <div className="tooltip">
```

Comment:
I'd like to ask you to rename these class to be SidebarButton-related -- ie, .SidebarButton-tooltip, .SidebarButton-tooltip .SidebarButton-tooltiptext.